src/components/table/store.ts

```
@@ -58,13 +70,20 @@ export const useTableStore = create<TableStore>()(
       set(state => {
         const filters = state.filters.map(obj => {
           if (obj.key === filterKey) {
-            if (obj.type !== FILTER_TYPE.DATE_RANGE) {
-              const values = (obj.values as string[]).filter(objValue => objValue !== value)
+            switch (obj.type) {
+              case FILTER_TYPE.DATE_RANGE: {
+              }
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)